Sources/PinLayout.swift

```diff
@@ -83,6 +83,21 @@ public class PinLayout<View: Layoutable> {
         }
         apply()
     }
+    
+    #if os(iOS)
```

Comment:
Detail, but you could move this block just below `public var layoutMargins: PEdgeInsets`, so the proprities are declared from the most useful property order? Thanks

Reply:
👍